CONTRIBUTING.md

```diff
@@ -2,7 +2,6 @@
 
 Thanks for you interest onto Sonata projects!
 
-Want to contribute? Please read the following documentation first:
-[project/CONTRIBUTING.md](project/CONTRIBUTING.md).
+Want to contribute? Please read the [following documentation](templates/project/CONTRIBUTING.md) first.
 
 This is the only documentation you have to refer.
```

Comment:
For bonus points:
```suggestion
This is the only documentation you have to refer to.
```

Reply:
That's not fair ! I get a mistake on something I didn't even write 😭

Reply:
For bonus points I said! You don't have to do it :P

Reply:
Done ;)